privaterelay/templates/profile.html

```diff
@@ -40,12 +40,14 @@ <h2 class="section-headline dashboard-headline">Manage Email Aliases</h2>
             <span class="remaining-aliases">
               You have <span class="num-remaining-aliases">{% remaining_free_aliases relay_addresses %}</span> remaining free aliases
             </span>
+            {% if not relay_addresses|length == 5 %}
```

Comment:
Let's keep using the `settings.MAX_NUM_BETA_ALIASES` instead of hard-coding the limit anywhere. I already added a django context processor that makes all of the `settings` values available in all templates, so it's this easy:

```suggestion
            {% if not relay_addresses|length == settings.MAX_NUM_BETA_ALIASES %}
```